new/PRS13180GabraSellase.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13180GabraSellase" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gabra Śǝllāse</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="SH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-10-07">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1"> ገብረ፡ ሥላሴ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra
+                        Śǝllāse</persName>
+                </person>
+                <listRelation>
+                                       <relation name="saws:isCopierOf" active="PRS13180GabraSellase"
+                        passive="BGV003#a2">
```

Comment:
I would simply write `BGV003` instead of `BGV003#a2` (which refers to the additional note only), and leave `BGV003#a2` in the `<desc/>` below by entering something like Gabra Śǝllāse is indicated as the scribe of the manuscript in `BGV003#a2`.

Reply:
Now I am confused - is he the scribe of the entire manuscript or only of a2? If he is the scribe of the entire manuscript, I would suggest to reformulate, if he is only the scribe of a2, I would keep #a2 in the main relation.

Reply:
No, he is only the scribe of the Additio as stated in the "desc element. Therefore I pointed to msitem BGV003#a2 but I thought one should now point to the whole ms abd than in the desc-element to the specific item. 
Shall I change it again? If yes,  Do I keep the "desc-element"?

Reply:
Ok, then it's my mistake. I unduly believed that he was the scribe of the entire manuscript and that this information was declared in Additio 2. I did not check the pictures, sorry. If he only wrote Additio 2, then @SusanneHummel's former statement was correct (`passive="BGV003#a2"`). Still, I would make this info unambiguous in the Desc element and state that he wrote that Additio only.